### ISSUE TITLE
Opening a repo without logging in doesn't draw the diagram 

### ISSUE THREAD

**Issue by ovrynn** [uvicgse/project-virtual-team-2#16]
Opening a local repository when you're not logged in doesn't draw the graph. It just says the application is 'Opening Local Repository' and never finishes the graph. You can still close the modal and it still shows the files that have been changed, but the graph is just missing.

Steps to reproduce:

Open the application
Click 'Continue without sign in'
Open a local repository
Actual:

![image](https://user-images.githubusercontent.com/22312351/57653671-3b73f280-7587-11e9-9b9b-6f7d43927fa4.png)

Expected:
Should still show this even if you're not logged in.
![image](https://user-images.githubusercontent.com/22312351/57653686-42026a00-7587-11e9-88c8-683e5ee0caa8.png)

**Comment by jbenning97** [comment]
Unable to reproduce, tested with different repositories; it must have been fixed.

**Comment by mdesco18** [comment]
@jbenning97 did you check the log to see if it signed you in with saved credentials

**Comment by jbenning97** [comment]
@mdesco18 There is nothing in the console that indicates that I am logged in. Should I be looking at a certain log message in particular? Are you able to reproduce?

**Comment by mdesco18** [comment]
> Should I be looking at a certain log message in particular?

Since I have signed in with the remember me checked even if i click continue without sign in then the first logged message is `button clicked. logging in with saved credentials`

**Comment by jbenning97** [comment]
> > Should I be looking at a certain log message in particular?
> 
> Since I have signed in with the remember me checked even if i click continue without sign in then the first logged message is `button clicked. logging in with saved credentials`

You're right. But now I uncheck the checkbox, remove my credentials, click 'continue withou...' and it still signs me in 🤦‍♂

**Comment by mdesco18** [comment]
We can fix this by including a button on the graph similar to team1 that calls refreshAll()
